cmd/httpclient/main.go

```diff
@@ -39,12 +40,9 @@ import (
 )
 
 var (
-	flagDNSUDPServer = flag.String(
-		"dns-udp-server", "1.1.1.1:53", "Server to use with -dns-transport udp",
-	)
-	flagDNSTransport = flag.String("dns-transport", "", "DNS transport to use")
-	flagSNI          = flag.String("sni", "", "Force specific SNI")
-	flagURL          = flag.String("url", "https://ooni.io/", "URL to fetch")
+	flagDNSServer = flag.String("dns-server", "system://", "Server to use")
```

Comment:
Same comment as above, I'd suggest to use `system:///` here since it's canonical